.cirrus.yml

```diff
@@ -54,6 +54,15 @@ global_task_template: &GLOBAL_TASK_TEMPLATE
 #  install_script:
 #    - choco install python --version=3.7.7 -y
 
+lint_task:
+  name: 'lint'
+  skip: $CIRRUS_REPO_FULL_NAME == "bitcoin-core/gui" && $CIRRUS_PR == ""  # No need to run on the read-only mirror, unless it is a PR.
+  container:
+    image: ubuntu:bionic
+  pip_cache:
+    folder: /tmp/pip_cache_dir
+  script: set -o errexit; source ./ci/lint/lint_run_all.sh
```

Comment:
Could run the linters with compute credits? The most frustrating thing is waiting for the linters to fail (or pass)


```
  # For faster CI feedback, immediately schedule the linters. https://cirrus-ci.org/pricing/#compute-credits
  use_compute_credits: $CIRRUS_REPO_FULL_NAME == 'bitcoin/bitcoin'